backend/schema.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS "users"(
+    "userId" VARCHAR(10) NOT NULL,
+    "userName" VARCHAR(40) NOT NULL,
+    "email" VARCHAR(255) NOT NULL,
+    "role" VARCHAR(255) NOT NULL,
+    "password" VARCHAR(100) NOT NULL,
+    "createdAt" TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+    "updatedAt" TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL
+);
+ALTER TABLE
+    "users" ADD PRIMARY KEY("userId");
+CREATE TABLE IF NOT EXISTS "companies"(
+    "companyId" VARCHAR(10) NOT NULL,
+    "companyName" VARCHAR(40) NOT NULL,
+    "address" VARCHAR(255) NOT NULL,
+    "createdAt" TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+    "updatedAt" TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL
+);
+ALTER TABLE
+    "companies" ADD PRIMARY KEY("companyId");
+CREATE TABLE IF NOT EXISTS "job_applications"(
+    "userId" VARCHAR(10) NOT NULL,
```

Comment:
This is foreign key to (now) `id` column of users table.
Please add the constraint and update accordingly.

Reply:
fixed